src/components/chips/js/chipsController.js

```diff
@@ -329,6 +329,7 @@ MdChipsCtrl.prototype.updateChipContents = function(chipIndex, chipContents){
   if(chipIndex >= 0 && chipIndex < this.items.length) {
     this.items[chipIndex] = chipContents;
     this.ngModelCtrl.$setDirty();
+    this.ngModelCtrl.$setViewValue(this.items.slice());
```

Comment:
Since this line is repeated 3-times - why not wrap it into a helper-function similar to validateModel() ?

Reply:
It seems there is already a function called `validateModel` and I think that `updateModel` describes better what does this statement do.

Reply:
But do we really need an helper function for just a single line of instruction?

On the other hand, if you meant to wrap into a function the two lines that are repeated

```
this.ngModelCtrl.$setDirty();
this.ngModelCtrl.$setViewValue(this.items.slice());
```
then I think we can remove `this.ngModelCtrl.$setDirty()`, since AFAIK `$setViewValue()` should already set the model as $dirty

Reply:
oh, I maybe choosed the wrong words. I meant something like the validateModel(), not name it this way.
But it is 3-times calling a function `$setViewValue` and chooses the a copy (`slice()`) from the inner value-cache (`this.items`). Maybe there is a small change there so the developer has to update now 3-lines instead of one. 
This is just an suggestion to wrap this in an extra function because I like to reduce repeated code and make it easier to maintain. But every developer has different workflows. 

PS.: if this helper-function could also reduce other duplications it is even better.